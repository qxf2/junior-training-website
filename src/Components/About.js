import React, { Component } from "react";
import Fade from "react-reveal";

class About extends Component {
  render() {
    return (
      <section id="about">
        <Fade duration={1000}>
          <div className="row">
            <div className="row">
              <h2>
                <a href="https://qxf2.com/">Qxf2 Services</a>
              </h2>
            </div>
          </div>
        </Fade>
      </section>
    );
  }
}

export default About;
