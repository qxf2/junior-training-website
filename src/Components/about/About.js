import React, { Component } from "react";
import Fade from "react-reveal";

class About extends Component {
  render() {
    return (
      <section id="about" data-testid="about" >
        <Fade duration={1000}>
          <div className="row">
              <h4>
                <a href="https://qxf2.com/">Qxf2 Services</a>
              </h4>
              <p>We are experienced testers working at Qxf2 Services. Qxf2 provides software testing services for startups. We produce a lot of open source material to help testers stay in touch with technical advancements in the field of testing. Qxf2 Services has been conducting a very successful internship program since 2016. Our interns have gone on to work at companies like Mphasis, Accenture, Temenos, Bosch, TCS, Qlik, StrikeTru, Surebot Technology Solutions, Infilect, Plaxonic Technologies and of course Qxf2 Services. Over time, we realized that the way we helped our interns bridge the gap between college and the industry could be applicable to a wider range of engineers. The approach, techniques and tools used in this training program are a result of our learning from our previous internships and our own industry experience.</p>
            </div>
        </Fade>
      </section>
    );
  }
}

export default About;
