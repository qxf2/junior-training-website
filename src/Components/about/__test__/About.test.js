import React from 'react';
import ReactDOM from 'react-dom';
import About from '../About';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<About/>, div);
});

it("renders about correctly", () => {
   const { getByTestId }  = render(<About/>)
   expect(getByTestId('about'));
});

it("About.js matches snapshot", () =>{
  const tree = renderer.create(<About/>).toJSON();
  expect(tree).toMatchSnapshot();
});