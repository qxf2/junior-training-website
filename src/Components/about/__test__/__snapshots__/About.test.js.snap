// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.js matches snapshot 1`] = `
<section
  data-testid="about"
  id="about"
>
  <div
    className="react-reveal row"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <h4>
      <a
        href="https://qxf2.com/"
      >
        Qxf2 Services
      </a>
    </h4>
    <p>
      We are experienced testers working at Qxf2 Services. Qxf2 provides software testing services for startups. We produce a lot of open source material to help testers stay in touch with technical advancements in the field of testing. We have been running a training program for senior QA engineers since 2019. Before launching this training program, we spent many years fine tuning our approach via many internships. Our interns have gone on to be placed at good companies soon after completing our internship program. The approach, techniques and tools used in this training program are a result of our learning from our previous internship and our own industry experience.
    </p>
  </div>
</section>
`;
